### ISSUE TITLE
val-data

### ISSUE THREAD

**Issue by llexieguo** [richard-peng-xia/MMed-RAG#29]
Hi, could anyone help understand what is '--val-data "/path/to/val.json' in the finetune_clip.sh needs to be?

**Comment by richard-peng-xia** [comment]
Hi @llexieguo, 

we only provide data for retrieval model training. Because the data in the validation set for the retriever does not matter, you only need to select a part of it from the same source dataset. Note that it cannot be repeated with the training set. 

You can see in the Issue #25 as well.
